### DISCUSSION TITLE
How to package "colour" with "Pyinstaller"?

### DISCUSSION THREAD

**Discussion by farhanalfin** [colour-science/colour#784] [Q&A] [answered]
I am working over a project using colour library, using python 3.9 32 bit and pycharm is working, but when I convert it to exe using pyinstaller the exe does not work. To debug I created a simple program, the exe of this program does not work 
my system is windows 10 64 
converting command pyinstaller simple_colour.py

the error
Application Error The Exception Unknown Software Exception 0xC06D007E

import colour
from PyQt5.QtWidgets import QApplication, QMainWindow, QLabel

class mainwindow(QMainWindow):
    def __init__(self):
        QMainWindow.__init__(self)
        
        self.label = QLabel("Colour module is loaded")
        self.setCentralWidget(self.label)

app = QApplication([])
window = mainwindow()
window.show()
app.exec_()

**Comment by MrLixm** [comment, marked as answer]
Hello, In my opnion it was not necessary to open a new issue for now but first answer to the replies you had on the first issue you opened here (even if it was on the colour-checker-detection repo) and tell if the suggested solution worked:
[https://github.com/colour-science/colour-checker-detection/discussions/16#discussioncomment-348455](https://github.com/colour-science/colour-checker-detection/discussions/16#discussioncomment-348455).

Cheers.
Liam.

**Reply by thomasmansencal** [reply]
Thanks @MrLixm!

@farhanalfin : This is a duplicate of https://github.com/colour-science/colour-checker-detection/discussions/16#discussioncomment-348455 as mentioned just above. Marking this one as answered!
